src/js/Drift.js

```diff
@@ -44,7 +44,8 @@ export default class Drift {
     // if `false`, it will switch to inline when `windowWidth <= inlinePane`
     const inlinePane = options["inlinePane"] || 375;
     // If `true`, touch events will trigger the zoom, like mouse events.
-    const handleTouch = options["handleTouch"] || true;
+    // const handleTouch = options["handleTouch"] || true;
+    const handleTouch = "handleTouch" in options ? options["handleTouch"] : true;
```

Comment:
We can ensure that `handleTouch` gets assigned a truthy value based on what's passed in by making the following change:
`const handleTouch = "handleTouch" in options ? !!options["handleTouch"] : true;`

The same should be done for `injectBaseStyles` on line 58